UnityProject/Assets/Scripts/Core/Chat/Chat.Process.cs

```diff
@@ -401,18 +411,18 @@ private static bool GhostValidationRejection(uint originator, ChatChannel channe
 		{
 			var getOrigin = NetworkIdentity.spawned[originator];
 			if (channels == ChatChannel.Local || channels == ChatChannel.Combat
-											  || channels == ChatChannel.Action)
+			                                  || channels == ChatChannel.Action)
 			{
-				LayerMask layerMask = LayerMask.GetMask("Walls", "Door Closed");
+				LayerMask layerMask = LayerMask.GetMask("Door Closed");
 				if (Vector2.Distance(getOrigin.transform.position,
-						PlayerManager.LocalPlayer.transform.position) > 14f)
+					PlayerManager.LocalPlayer.transform.position) > 14f)
 				{
 					return true;
 				}
 				else
 				{
-					if (Physics2D.Linecast(getOrigin.transform.position,
-						PlayerManager.LocalPlayer.transform.position, layerMask))
+					if (MatrixManager.RayCast(getOrigin.transform.position, Vector2.zero, 0, LayerTypeSelection.Walls,
+						layerMask, PlayerManager.LocalPlayer.transform.position).ItHit);
```

Comment:
Codacy found an issue: [Remove this empty statement.](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6319358)